site/graphql-js/Tutorial-Mutations.md

```diff
@@ -8,25 +8,28 @@ next: /graphql-js/authentication-and-express-middleware/
 
 If you have an API endpoint that alters data, like inserting data into a database or altering data already in a database, you should make this endpoint a `Mutation` rather than a `Query`. This is a simple as making the API endpoint part of the top-level `Mutation` type instead of the top-level `Query` type.
 
-Let's say we have a “message of the day” server, where anyone can update the message of the day along with the author of the message, and anyone can read the current one. The GraphQL schema for this is simply:
+Let's say we have a “message of the day” server, where anyone can update the message of the day, and anyone can read the current one. The GraphQL schema for this is simply:
 
 ```javascript
 type Mutation {
-  setMessage(message: String)
+  setMessage(message: String): Boolean
```

Comment:
More typically graphql mutations return the thing they were mutating so clients can be aware of anything the server did (maybe you lost a race condition with set-message with someone else, or maybe the server did some post-processing) but also as a convenient mechanism to keep a client cache up to date.

That's perhaps overkill here, and illustrating that it's okay to return a boolean is good, but something like this should probably be mentioned here in reference to the fuller example you have below